### DISCUSSION TITLE
Help and documentation

### DISCUSSION THREAD

**Discussion by pkphilip** [Embarcadero/DelphiFMX4Python#27] [Q&A]
I would really appreciate a link to the documentation for the FMX controls. Can this link be added into the project's Readme file? or from the Wiki?

**Comment by Priyatham10** [comment]
Thank you for the request @pkphilip . We're currently working on ways on documentation. The way we're thinking is to port the documentation from the existing Delphi docs to Python libraries. We'll revert back as we made some considerable progress on it.

**Reply by pkphilip** [reply]
Thank you so much for the update. May I also recommend that you include examples with screenshots as many people are not familiar with what is possible with Delphi and VCL.. and that would extend to the FMX controls as well

**Reply by Priyatham10** [reply]
Thank you for the new thinking. That is also in the plan to include cool app examples built with the libraries on README as a Gallery section. 

We're building such cool app examples now and then we shall add them. We'll also accept the app screenshots from the community as we find them cool to put them in the Gallery. Please try to come up with such more new ideas as you get them.

**Comment by lmbelo** [comment]
Docs are now available. See the image bellow:

![Screenshot 2023-03-07 at 18 02 27](https://user-images.githubusercontent.com/8376898/223551866-efe0d41d-cb25-49cd-a020-bab1f2111852.png)
